hack/ocm-minikube-ramen.sh

```diff
@@ -145,27 +145,28 @@ ramen_undeploy_spoke()
 	ramen_undeploy $1 $2 dr-cluster
 }
 exit_stack_push unset -f ramen_undeploy_spoke
-ocm_ramen_samples_git_ref=main
+ocm_ramen_samples_git_ref=${ocm_ramen_samples_git_ref-main}
+ocm_ramen_samples_git_user=${ocm_ramen_samples_git_user-ramendr}
```

Comment:
I don't know what this component of a github address is called.  In some cases it is a user, in others an organization.   In #207 I called it a `github_name`.  `user` seems fine, but I think it is `github` one, not a `git` one.

Reply:
Good point. These are the components of a repo's URL
- git://host.xz[:port]/path/to/repo.git/
- http[s]://host.xz[:port]/path/to/repo.git/

Hence, it may be appropriate to name it as "path": ocm_ramen_samples_git_path or ocm_ramen_samples_git_path_to.